components/query-controls/index.js

```diff
@@ -26,11 +22,11 @@ export default function QueryPanel( {
 	minItems = DEFAULT_MIN_ITEMS,
 	onCategoryChange,
 	onNumberOfItemsChange,
-	onOrderChange = noop,
-	onOrderByChange = noop,
+	onOrderChange,
+	onOrderByChange,
 } ) {
 	return [
-		( onOrderChange || onOrderByChange ) && (
+		( onOrderChange && onOrderByChange ) && (
 			<SelectControl
 				key="query-panel-select"
```

Comment:
There are 3 `key` props that still reference panel. All of them should start now with: `query-controls-*`.

Reply:
Nice catch keys were updated.